package.json

```diff
@@ -73,7 +73,7 @@
     "react-toastify": "^6.0.9",
     "sanitize-html": "^1.20.1",
     "serialize-javascript": "^3.1.0",
-    "typescript": "3.9.3",
+    "typescript": "4.1.2",
```

Comment:
Got this: Maybe typescript-eslint should be updated also? Preferably I would add version bumps in separate pr.
`WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
You may find that it works just fine, or you may not.
SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <4.1.0
YOUR TYPESCRIPT VERSION: 4.1.2
`

Reply:
downgraded back for now